src/typeahead/typeahead.js

```diff
@@ -150,7 +151,7 @@ angular.module('ui.bootstrap.typeahead', ['ui.bootstrap.position', 'ui.bootstrap
 
         hasFocus = true;
 
-        if (inputValue && inputValue.length >= minSearch) {
+        if (minSearch === 0 || inputValue && inputValue.length >= minSearch) {
```

Comment:
Is there check for invalid inputs here - strings, numbers as strings, negative numbers ...?